src/localize/languages/lu.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
you also have to map the values in https://github.com/pkissling/clock-weather-card/blob/master/src/localize/localize.ts and please amend the supported languages in the README.md

Reply:
Thanks for pointing it out. It's done now.